pkgx.rb

```diff
@@ -22,35 +22,27 @@ class Pkgx < Formula
   end
 
   depends_on "deno" => :build
-  depends_on "unzip" => :build # deno >=1.39.1 uses unzip when remote-fetching their compilable runtime
+  depends_on "rust" => :build
+  depends_on "sqlite" => :linux
 
   def install
-    system "deno", "task", "compile"
+    if File.file? "Cargo.toml"
+      system "cargo", "build", "--release"
```

Comment:
This will need to be moved (or bin.install will need the path target/release/pkgx, right?)

Reply:
tru dat